Additions related to dating and scheduling posts. (#2853)
GoogleChrome/web.dev: src/site/content/en/handbook/markup-post-codelab/index.md

* `title`: The title for your piece.
* `subhead`: The subtitle for your piece. Only required for posts.
* `authors`: The [slug(s)](/handbook/contributor-profile) for the author(s) of the piece.

> * `scheduled` Set to `true` to schedule a post for a future date. Posts will deploy at 7am PST / 15:00 UTC. Example: A post with `date: 2050-01-01`, `scheduled: true`, will go live at 7am PST, January 1st, 2050. If you don't use the scheduled flag then setting a future date has no effect. Only use this field un unpublished posts. Doing so an a published post will hide it from the site until the provided `updated` value.

Reviewer: ```suggestion
* `scheduled` Set to `true` to schedule a post for a future date. Posts will deploy at 7am PST / 15:00 UTC. Example: A post with `date: 2050-01-01`, `scheduled: true`, will go live at 7am PST, January 1st, 2050. If you don't use the scheduled flag then setting a future date has no effect. Only use this field on unpublished posts. Doing so on a published post will hide it from the site until the provided `updated` value.
```
